WebUI/Areas/Identity/Pages/Account/Register.cshtml

```diff
@@ -5,6 +5,7 @@
 }
 
 <h1>@ViewData["Title"]</h1>
+<p style="color:red">@ViewData["error"]</p>
```

Comment:
```suggestion
<p class="text-danger">@ViewData["error"]</p>
```
I would recommend using this instead of the inline style rule just to maintain consistency with our Bootstrap styling.